webserver/static/scripts/datasets/class-viewer.js

```diff
@@ -1,3 +1,4 @@
+var PropTypes = require('prop-types');
```

Comment:
we've started using typescript in LB to solve the typing problem, fyi. Maybe there's an incentive to stay consistent here.

Reply:
Not right now, obviously, but maybe as a end goal. Maybe new react components could be in typescript.

Reply:
yeah, some more cleanup of these scripts and potentially moving to ts is an option here. For now I'm just getting it working with new react.